add-on/src/lib/state.js

```diff
@@ -40,8 +41,15 @@ export function initState (options, overrides) {
       return false
     }
   }
-  // TODO state.connected ~= state.peerCount > 0
-  // TODO state.nodeActive ~= API is online,eg. state.peerCount > offlinePeerCount
+  // TODO refactor this into a class. It's getting too big and messy.
+  Object.defineProperty(state, 'connected', {
+    // TODO: make quick fetch to confirm it works?
+    get: function () { return this.peerCount > offlinePeerCount + 1 }
+  })
```

Comment:
This `+ 1` will cause bugs when your local  node is running, but your internet connection goes down.  Imagine you opened something, it got cached on your IPFS node, then you shut down your laptop, go to a place without iternet and you opened laptop and restarted browser and  there is a tab with localhost gateway URL.

User should be able to  browse IPFS resources cached on their IPFS node when WAN is down, but due to `peerCount > 0` requirement here Companion will show recovery page, forbidding them from leveraging IPFS cache.

Mind removing this and only triggering recovery when `peerCount === offlinePeerCount`?